File: A's/cpp/785A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n; cin >> n;
    int faces = 0;
    while (n--) {
        string s; cin >> s;
        transform(s.begin(), s.end(), s.begin(), ::tolower);
        if (s == "tetrahedron") faces += 4;
        else if (s == "cube") faces += 6;
        else if (s == "octahedron") faces += 8;
        else if (s == "dodecahedron") faces += 12;
        else if (s == "icosahedron") faces += 20;
    }
    cout << faces << endl;
    
    return 0;
}

File: A's/py/785A.py
def main() -> None:
    n: int = int(input())
    faces: int = 0
    while n:
        s: str = str(input())
        s = s.lower()
        if s == "tetrahedron": faces += 4
        elif s == "cube": faces += 6
        elif s == "octahedron": faces += 8
        elif s == "dodecahedron": faces += 12
        elif s == "icosahedron": faces += 20
        n -= 1
    print(faces)

if __name__ == "__main__":
    main()